docs/deployment-guide.md

```diff
@@ -0,0 +1,396 @@
+## WonderTix Deployment Guide
+
+Created by Jacob Bentley (jacobpbentley@gmail.com)
+
+![alt text](doc-images/image-11.png)
+
+### Introduction
+
+This document provides a walkthrough of the WonderTix deployment on Google Cloud Platform (GCP). Its purpose is to describe each of the main components and provide some instructions for common or important tasks. Where appropriate, additional details can be found in separate, linked documents.
+
+### The Big Three
+
+The three most important services in our GCP architecture are Cloud Build, Cloud Run, and Cloud SQL. We’ll go through each of them in turn, or you can click one of the above links to jump straight to that section.
+
+### Cloud Build
+
+#### Triggers and workflows
+
+Cloud Build is the link between our GitHub repo and the rest of GCP. It’s how our code gets packaged up into a deployable form and sent over to Cloud Run, which does the actual deploying. It does this primarily through the use of triggers (pictured below).
+
+![alt text](doc-images/doc-images/image-12.png)
```

Comment:
This path should just be `doc-images/image-12.png`.

Reply:
Ah good catch, that was me being reckless with find and replace https://github.com/WonderTix/WonderTix/pull/635/commits/75cd7ffeb075c08404495a127fbdc813b08b8ca4